src/datafeeds/DataFeed.sol

```diff
@@ -78,12 +80,7 @@ contract DataFeed is DataFeedInterface, AssetRegistrar {
     /// @dev Uses an efficient bulk call
     /// @param ofAsset Asset for which data history should be returned
     /// @param withStartId Index at which history should be started, this is due to the limitation of non dynamic array size returns
-    /**
-    @return {
-      "timestampArray": "Array of timestamps",
-      "priceArray": "Array of prices"
-    }
-    */
+    /// @return Arrays of timestamps and prices of ofAsset
```

Comment:
here would keep
```
   @return {
-      "timestampArray": "Array of timestamps",
-      "priceArray": "Array of prices"
-    }
-    */
```

doxity needs this style of formatting whenever multiple return values should be documented